readme.md

```diff
@@ -1157,6 +1165,14 @@ echo $faker->name; // 'Sr. Luis Adriano Sepúlveda Filho'
 echo $faker->cpf;        // '145.343.345-76'
 echo $faker->cpf(false); // '45623467866'
 echo $faker->rg;         // '84.405.736-3'
+
+// More functions
+echo $faker->titleMale; // 'Sr.'
+echo $faker->titleFemale; // 'Srta.'
+echo $faker->suffix; // 'Jr.'
+echo $faker->firstNameMale; // 'Felipe'
```

Comment:
please don't document as locale-specific a formatter which is common to all locales